docs/integration-tests.md

```diff
@@ -0,0 +1,74 @@
+## Integration Tests
+
+Debugger.html integration test are end-to-end tests that simulate a user debugging an application.
```

Comment:
Looks fine to me, but I wouldn't work on this too much more right now. This is most unknown thing right now. Personally I'm not sure we will end up using cypress, but we need to look into other things before making that decision. As this is in such flux, I wouldn't encourage people to write these tests right now.